### PR TITLE
Preparing opent2t-cloud package for hubcontroller class distribution

### PR DESCRIPTION
**Pull request by shalinijoshi19**
- Updated Readme.md
- Updated package.json to re-define package name to be "opent2t-cloud" to be consistent with the repo name.
- Including only hubController.js and its dependencies at the moment in the package
- Updating hubConfig.js to include other supported hubs today by OpenT2T.

### CODE REVIEW

File: node/hubsConfig.js [hunk 1]
```diff
@@ -8,7 +8,22 @@ var hubs = [
...
+    { 
+        'id': "nestHub",
+        'name': 'Nest',
+        'translator': 'opent2t-translator-com-nest-hub'
+    },
+    { 
+        'id': "smartthingsHub",
+        'name': 'SmartThings',
+        'translator': 'opent2t-translator-com-smartthings-hub'
+    },
```
Comment (arjun-msft) [inline node/hubsConfig.js]: add hue too

**Comment by arjun-msft** [comment]
>     "request": "*"

since we are limiting to hubController, a lot of these aren't necessary

---
Refers to: node/package.json:33 in a38b5e8. [](commit_id = a38b5e810723eda49d8ca0bc7fd85aca78d73b42, deletion_comment = False)

File: node/publish.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+if (process.argv[2] === "prepublish") {
+    prepublish();
+} else if (process.argv[2] === "postpublish") {
+    postpublish();
+}
+
+function prepublish() {
+    // Explicitly copy readme.md from root to this current folder
+    // so the npm publish can pick this up and include it in the package.
```
Comment (arjun-msft) [inline node/publish.js]: it this necessary for this?  isn't everything done from the root.

**Comment by arjun-msft** [comment]
:shipit:

File: node/hubsConfig.js (hunk 1 above)
Comment (avneet723) [inline node/hubsConfig.js]: extra comma

Comment (shalinijoshi19) [inline node/hubsConfig.js]: Done thanks

Comment (shalinijoshi19) [inline node/hubsConfig.js]: I believe that's allowed in .json syntax but removed none-the-less. TS linters actually want the extra comma !

File: node/publish.js (hunk 2 above)
Comment (shalinijoshi19) [inline node/publish.js]: The package.json is under node. and based on the opent2t experience it doesn't pick up the readme automagically if it's not in the same folder as root.

**Pull request merged**